sites/partners/src/components/listings/PaperListingForm/sections/RankingsAndResults.tsx

```diff
@@ -273,6 +249,28 @@ const RankingsAndResults = ({ listing, isAdmin }: RankingsAndResultsProps) => {
             </Grid.Row>
           </>
         )}
+        <Grid.Row columns={2} className={"flex items-center"}>
```

Comment:
I think this logic should be moved to `sites/partners/src/components/listings/PaperListingForm/sections/ApplicationDates.tsx`. This question gets hidden when Lottery is selected and is no longer tied to FCFS so should be with the actual date and time fields at the bottom

Reply:
@mcgarrye I think that would be a design change and Em has been checking out the UX of the toggled on fields already, I think this is a great point that would be part of separate work

Reply:
Very fair. Still should appear when lottery is selected though if we aren't moving it

Reply:
Very sorry for the back and forth on the AC here @KrissDrawing! After chatting w Sarah and Em, the desired behavior is to remove the `Is there an application due date?` question and always have the due date field be visible and enabled.

Reply:
Not a problem, if checks pass, it should be ready to review